elvis.config

```diff
@@ -3,42 +3,41 @@
    elvis,
    [
     {config,
-     [#{dirs => ["src", "test"],
+     [#{dirs => ["src"],
```

Comment:
If you're about to remove the test folder from here, please add another config group to check it.